app/src/main/java/com/pvp/app/service/ExerciseServiceImpl.kt

```diff
@@ -120,4 +120,10 @@ class ExerciseServiceImpl @Inject constructor(
                     activity != SportActivity.Other
         }
     }
+
+    override suspend fun getMostFrequentActivity(): SportActivity {
```

Comment:
Ah, that is being done, what I mentioned above... Comment could be improved in the interface to let user know this 😁